src/main/java/frc/robot/subsystems/hood/HoodConstants.java

```diff
@@ -43,5 +54,22 @@ public static void initialize(Constants.Mode mode) {
                 kV.initDefault(0.0);
                 kA.initDefault(0.0);
         }
+        motorConfiguration
+                .withFeedback(new FeedbackConfigs().withSensorToMechanismRatio(GEAR_RATIO))
+                .withSlot0(
+                        new Slot0Configs()
+                                .withKP(HoodConstants.kP.get())
+                                .withKI(HoodConstants.kI.get())
+                                .withKD(HoodConstants.kD.get())
+                                .withKS(HoodConstants.kS.get())
+                                .withKV(HoodConstants.kV.get())
+                                .withKA(HoodConstants.kA.get())
+                                .withKG(HoodConstants.kG.get()))
+                .withMotorOutput(new MotorOutputConfigs().withInverted(INVERTED_VALUE))
+                .CurrentLimits
+                .withStatorCurrentLimitEnable(true)
+                .withSupplyCurrentLimitEnable(true)
+                .withStatorCurrentLimit(CURRENT_LIMIT)
+                .withSupplyCurrentLimit(CURRENT_LIMIT);
```

Comment:
You should do it how it's on lines 70-71 -- if the constant needs to be configured on the fly, use a tunable number. Unless, write the value directly here.

Reply:
f37b0b7